.vib/consul/cypress/cypress/integration/consul_spec.js

```diff
@@ -0,0 +1,68 @@
+/// <reference types="cypress" />
+
+import { random } from './utils';
+
+it('shows all instances of services', () => {
+  cy.visit('/ui/datacenter1/services');
+  cy.contains('Services');
+  cy.get('.linkable').contains('consul');
+  cy.get('.detail span').contains('3 instances');
```

Comment:
I believe that having 3 instances is entirely dependent on installation parameters (`replicaCount`). I would add it to the `runtime_params` & `cypress.json` env var section, as we are actively checking this value